llm/llama.go

```diff
@@ -7,6 +7,9 @@ package llm
 #cgo darwin CPPFLAGS:  -DGGML_USE_ACCELERATE
 #cgo darwin,arm64 CPPFLAGS: -DGGML_USE_METAL -DGGML_METAL_NDEBUG
 #cgo darwin LDFLAGS: -framework Accelerate -framework Foundation -framework Metal -framework MetalKit -framework MetalPerformanceShaders
+#cgo linux,cuda CFLAGS: -DGGML_USE_CUBLAS -I/usr/local/cuda/include
+#cgo linux,cuda CXXFLAGS: -DGGML_USE_CUBLAS -I/usr/local/cuda/include
```

Comment:
These should really go into `CPPFLAGS` which are intended for the "C Pre Processor". `CPPFLAGS` get passed to both `CC`, the C compiler, as well as `CXX`, the C++ compiler.